arbitrum-client/src/conversion.rs

```diff
@@ -381,6 +439,17 @@ pub fn pk_to_u256s(pk: &PublicSigningKey) -> Result<[AlloyU256; NUM_SCALARS_PK],
 // | CONVERSION UTILITIES |
 // ------------------------
 
+/// Convert a `BigUint` to an `Address`
+pub fn biguint_to_address(biguint: &BigUint) -> Result<Address, ConversionError> {
```

Comment:
tiny nit but can we apply this elsewhere that it should be used, e.g. `to_contract_external_transfer`?